tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/OnboardMember.kt

```diff
@@ -65,19 +67,19 @@ class OnBoardMember : Runnable, BaseOnboard() {
         File(File(File(File(System.getProperty("user.home")), ".corda"), "gp"), "groupPolicy.json")
 
     @Option(
-        names = ["--cpi-hash"],
-        description = ["The CPI hash (Use either cpb-file or cpi-hash)."]
+        names = ["--cpi-hash", "-h"],
+        description = ["The CPI hash of a previously uploaded CPI (use either --cpb-file or --cpi-hash)."]
     )
     var cpiHash: String? = null
 
     @Option(
-        names = ["--pre-auth-token"],
+        names = ["--pre-auth-token", "-a"],
```

Comment:
```suggestion
        names = ["--pre-auth-token", "-t"],
```
or would this clash with anything else?

Reply:
That would clash with `RestCommand`'s target option